hackclub.com.yaml

```diff
@@ -1904,4 +1904,10 @@ zwp:
   type: A
   value: 172.99.233.117
   
+hack-a-way.hackclub.dom:
+-ttl: 1
+ type:A
+ value:127.0.0.1:5500/index.html.
```

Comment:
This PR isn't correct for two reasons:

1. the root domain shouldn't be included in the target
2. the website must be publicly hosted, not locally. also don't include the `index.html`